mcs/class/System/System/MonoExeLocator.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace System {
+
+	public static class MonoExeLocator {
+
+		public static string GacPath { get; }
```

Comment:
This is a new C#6 feature that might not be available when we bootstrap with an older Mono. Better to use `{ get; private set; }`